StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -56,16 +56,26 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
 
     UpdateScene = &DefaultUpdateScene;
 
-    //sceneManager->SetScene(JacobsTestScene(sceneManager));
-    sceneManager->SetScene(LeosTestScene(sceneManager));
-    //sceneManager->SetScene(LeosBounceScene(sceneManager));
-    //sceneManager->SetScene(TimScene(sceneManager));
-    //sceneManager->SetScene(JockesTestScene(sceneManager));
-    //sceneManager->SetScene(FloppipTestScene(sceneManager));
-    //sceneManager->SetScene(FredriksTestScene(sceneManager));
-    //sceneManager->SetScene(WilliamsTestScene(sceneManager));
-    //sceneManager->SetScene(BjornsTestScene(sceneManager));
-    //sceneManager->SetScene(AndresTestScene(sceneManager));
+    //Scene* jacobScene = JacobsTestScene(sceneManager);
+    Scene* leoScene = LeosTestScene(sceneManager);
+    //Scene* leoBounceScene = LeosBounceScene(sceneManager);
+    //Scene* timScene = TimScene(sceneManager);
+    //Scene* jockeScene = JockesTestScene(sceneManager);
+    Scene* filipScene = FloppipTestScene(sceneManager);
+    //Scene* fredrikScene = FredriksTestScene(sceneManager);
+    //Scene* williamScene = WilliamsTestScene(sceneManager);
+    //Scene* bjornScene = BjornsTestScene(sceneManager);
+    //Scene* antonScene = AntonTestScene(sceneManager);
+    //Scene* andresScene = AndresTestScene(sceneManager);
+
+    // Load Scenes *** Unload with UnloadScene()
```

Comment:
Should this still be here?

Reply:
SetScenes(2, activeScenes) calls loadscene. But just showing that you can LoadScenes. For easier to understand code I would say always write LoadScene() when changing scene to another. (Sandbox is just testing atm, not really changing a level with setscene).